src/test/resources/alma-fix/990053976760206441.json

```diff
@@ -201,7 +201,7 @@
       },
       "id" : "http://d-nb.info/gnd/4045977-9",
       "gndIdentifier" : "4045977-9",
-      "altLabel" : [ "Physik", "Physik", "Physikdidaktik", "Physik", "Physik", "Physikdidaktik", "Physik", "Physik", "Physikdidaktik" ]
+      "altLabel" : [ "Physik / Didaktik", "Physik / Unterricht", "Physikdidaktik", "Physik / Didaktik", "Physik / Unterricht", "Physikdidaktik", "Physik / Didaktik", "Physik / Unterricht", "Physikdidaktik" ]
```

Comment:
There are redundant entries. Could you make them unique?

Reply:
ah I found the reason for this error, "Physikunterricht" GND-040459772 is part of multiple Schlagwortfolgen and has multiple gnd enrichment entries, therefore the enrichment process adds multiple elements.
=> so make them uniq would be smart +1